clients/githubrepo/repo_test.go

```diff
@@ -78,11 +81,25 @@ func TestRepoURL_IsValid(t *testing.T) {
 			inputURL: "https://github.com/foo/kubeflow",
 			wantErr:  false,
 		},
+		{
+			name: "Enterprise github repository with host",
+			expected: repoURL{
+				host:  "github.corp.com",
+				owner: "corpfoo",
+				repo:  "kubeflow",
+			},
+			inputURL: "https://github.corp.com/corpfoo/kubeflow",
+			wantErr:  false,
+			ghHost:   true,
+		},
 	}
 	for _, tt := range tests {
 		tt := tt // Re-initializing variable so it is not changed while executing the closure below
 		t.Run(tt.name, func(t *testing.T) {
-			t.Parallel()
+			if tt.ghHost {
+				os.Setenv("GH_HOST", "github.corp.com")
```

Comment:
Should unset this after the test is over (see: https://beta.pkg.go.dev/testing#T.Cleanup).

Reply:
To keep things consistent with other test, addressing this with `t.Setenv` which auto cleanup